actors/stage/src/main/java/com/ea/orbit/actors/runtime/Execution.java

```diff
@@ -493,6 +492,10 @@ public Activation(final ReferenceEntry entry, final Object instance)
         }
     }
 
+    public void addStickyHeaders(Collection<String> stickyHeaders)
```

Comment:
No setter so developer doesn't accidently remove the transaction id sticky header.